x-pack/solutions/security/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/prebuilt_rule_upgrade_without_preview.md

```diff
@@ -798,3 +808,166 @@ And user opens the Rule Updates table
+And the upgraded prebuilt rule should be removed from the table
+And all customizable rule fields should be equal to the target version
+
+<customization_state> = customized | not customized
+```
+
+#### **Scenario: Multiple selected rules are upgraded to target versions when upgraded via upgrade table on insufficient license**
+
+**Automation**: 1 e2e test with a mock rule.
+
+```Gherkin
+Given a Kibana instance running under an insufficient license
+And multiple outdated prebuilt rules (new versions are available for them)
+When user is on the Rule Updates table
+And selects multiple rules
+And clicks a button to upgrade selected rules
+Then success message should be displayed after upgrade
+And all customizable fields of each upgraded rule should be equal to their respective target versions
+```
+
+#### **Scenario: All rules are upgraded to target versions when upgraded via upgrade table on insufficient license**
+
+**Automation**: 1 e2e test with a mock rule.
+
+```Gherkin
+Given a Kibana instance running under an insufficient license
+And multiple outdated prebuilt rules (a new version is available for them)
+When user is on the Rule Updates table
+And upgrades all rules at once using "Upgrade all" button
+Then success message should be displayed after upgrade
+And all customizable fields of each upgraded rule should be equal to their respective target versions
+```
+
+#### **Scenario: User can NOT filter by customization state in upgrade table when license is insufficient**
+
+**Automation**: 1 e2e test with a mock rule.
+
+```Gherkin
+Given a Kibana instance running under an insufficient license
+And there's at least one prebuilt rule available for upgrade
+When user is on the Rule Updates table
+Then user should NOT see UI elements to filter rules by customization state (Modified/Unmodified)
+```
+
+#### **Scenario: User can NOT see whether a rule has conflicts in upgrade table when license is insufficient**
+
+**Automation**: 1 e2e test with 2 mock rules.
+
+```Gherkin
+Given a Kibana instance running under an insufficient license
+And an installed prebuilt rule
+And this rule is outdated (a new version is available for this rule)
+And this rule has a <conflict_type> conflict
+When user is on the Rule Updates table
+Then user should NOT see any information about conflicts in the upgrade table
+
+Examples:
+<conflict_type> = solvable | non-solvable
+```
+
+#### **Scenario: User is NOT forced to review rule changes before upgrading when license is insufficient**
+
+**Automation**: 1 e2e test with 2 mock rules.
+
+```Gherkin
+Given a Kibana instance running under an insufficient license
+And an installed prebuilt rule
+And this rule is outdated (a new version is available for this rule)
+And this rule has a <conflict_type> conflict
+When user is on the Rule Updates table
+Then user should see an upgrade action available for this rule
+And user should be able to upgrade this rule without reviewing the changes
```

Comment:
It's interesting that we don't display any warnings in case of customized prebuilt rules. In a similar scenario for Prebuilt Rule Upgrade Flyout we display a warning message.